tests/unit/test_install.py

```diff
@@ -1638,3 +1650,23 @@ def test_extract_major_minor_versions():
 
     version_string3 = "should not match"
     assert extract_major_minor(version_string3) is None
+
+
+def test_account_installer(ws):
+    acc = create_autospec(AccountClient)
+    acc.workspaces.list.return_value = [Workspace(workspace_id=123), Workspace(workspace_id=456)]
```

Comment:
Aren't missing workspace names causing problems?

Reply:
the workspace names are only used for the prompt message, so the lack of it does not cause issue